plugins/permission-common/api-report.md

```diff
@@ -56,6 +44,11 @@ export enum AuthorizeResult {
 // @public
 export type BasicPermission = PermissionBase<'basic', {}>;
 
+// @public
+export type Batch<T> = {
```

Comment:
I'm feeling `Batch` is a bit too generic tbh, same thing with `Identified`. Mostly concerned with editor autocomplete, but also to some extend imports. Perhaps we can get "permission" or smth in there somehow, `PermissionMessageBatch`?

Reply:
Fair comment! Happy to rename both of these.